modules/api/current.js

```diff
@@ -210,7 +210,7 @@ module.exports = {
       if (!this.isScenarioBasedStatistics) return;
     }
 
-    const { name: scenarioName } = scenario;
+    const scenarioName = this.storage.getPickle(pickleId).name;
```

Comment:
@radekvonboktor  Hi, there!
[gherkin](https://github.com/gherking/gherkin-ast) creates ast.
And every AST node such as pickle in our case, associates with the scenario through ASTNodeID
I mean, that pickle and scenario are almost the same entities.
<img width="665" alt="Screenshot 2023-09-27 at 12 18 04" src="https://github.com/reportportal/agent-js-cucumber/assets/84175555/1fcade83-fd94-437e-94a3-0ed98d08db9c">

And as I can see, dynamic scenario naming already works fine.
Maybe I missed something. Could you please provide an example, where the scenario naming does not contain the value of the variable?